### PR TITLE
Use absolute urls for link which spans to different site

### PR DESCRIPTION
**Pull request by czpython**

### CODE REVIEW

File: aldryn_bootstrap3/model_fields.py [hunk 1]
```diff
@@ -160,8 +161,13 @@ class Meta:
...
 
     def get_link_url(self):
-        if self.link_page_id:
-            link = self.link_page.get_absolute_url()
+        if self.link_page:
+            ref_page = self.link_page
+            link = ref_page.get_absolute_url()
+
+            if ref_page.site_id != getattr(self.page, 'site_id', None):
+                ref_site = Site.objects._get_site_by_id(ref_page.site_id)
```
Comment (stefanfoulis) [inline aldryn_bootstrap3/model_fields.py]: using a private method?

Comment (czpython) [inline aldryn_bootstrap3/model_fields.py]: yes.. it saves a query.. otherwise a query is made for every plugin.

**Comment by FinalAngel** [comment]
@czpython can we merge?

**Comment by FinalAngel** [comment]
Fixes https://github.com/aldryn/aldryn-bootstrap3/issues/143

**Pull request merged**